.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,24 @@
+name: Bug
+description: File a bug report
+body:
+  - type: markdown
+    attributes:
+      value: "Hello from the Apollo Server team! Hopefully we can help resolve your issue. To increase the chances of us being able to help, please take the time to fill out the form as completely as possible. A minimal, runnable reproduction is the best way to get us to help you quickly and in many cases we simply cannot help without one."
+  - type: textarea
+    attributes:
+      label: Issue Description
+      description: Describe the issue you are experiencing
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: Link to Reproduction
+      description: GitHub or CodeSandbox link with runnable reproduction
```

Comment:
```suggestion
      description: GitHub or CodeSandbox link with runnable reproduction. Make sure this includes everything necessary (package.json, tsconfig.json, etc) so we don't have to guess anything!
```

Reply:
I don't mind the additional clarity, we'll see how it looks on the form!